### DISCUSSION TITLE
How to: Confirm Winlogbeat configuration does NOT use the Elastic Sysmon module

### DISCUSSION THREAD

**Discussion by wildlyunder** [Security-Onion-Solutions/securityonion#5935] [Unsupported Versions] [answered]
Hi,

How do I confirm that my Winlogbeat configuration does NOT use the Elastic Sysmon module

**Comment by dougburks** [comment, marked as answer]
If you follow all instructions at https://docs.securityonion.net/en/2.3/beats.html#winlogbeat without any additional customization, then your Winlogbeat configuration should be correct.
